src/Client/Stripe.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace MatchBot\Client;
+
+use MatchBot\Domain\Currency;
+use MatchBot\Domain\Money;
+use MatchBot\Domain\StripeCustomerId;
+use Stripe\StripeClient;
+
+class Stripe
+{
+    public function __construct(private StripeClient $stripeClient)
+    {
+    }
+
+    public function fetchBalance(StripeCustomerId $stripeCustomerId, Currency $currency): Money
```

Comment:
Implementation here is largely copied from https://github.com/thebiggive/identity/blob/453cfc343ebabdd738a790d67d1f126a888983f2/src/Application/Actions/Person/Get.php#L94